react/features/polls/components/AddPollItem.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import React, { Component } from 'react';
+
+type Props = {
+    addItemHandler: Function
+};
+
+/**
+ * Input component for adding poll options.
+ */
+class AddPollItem extends Component<Props, *> {
+
+    /**
+     * Constructor.
+     *
+     * @inheritdoc
+     */
+    constructor(props: Props) {
+        super(props);
+
+        this._onClick = this._onClick.bind(this);
+    }
+
+    /**
+     * Component render method.
+     *
+     * @inheritdoc
+     */
+    render() {
+        return (
+            <div
+                className = { 'pollNewChoiceButtonContainer' } >
+                <button
+                    id = { 'pollNewChoiceButton' }
+                    onClick = { this._onClick } >
+                    +
```

Comment:
I'm not certain about the UX of having a + button. I think, not speaking for the team, it would be preferred if fields were automatically added as the person entered choices. That reduces a click and maybe reduces what the user needs to know to use the feature.

Reply:
Fixed when user press enter as typing a choice, creates a new choice automatically.